snmp/tests/common.py

```diff
@@ -159,7 +159,7 @@
     {
         'MIB': "IF-MIB",
         'table': "noIdeaWhatIAmDoingHere",
-        'symbols': ["ifInOctets", "ifOutOctets"],
+        'symbols': ["ImWrong", "MeToo"],
```

Comment:
Since the `table` parameter is not used anymore for querying, it's the actual symbols that have to be wrong for the test for invalid metrics to pass.